test/formatters/apheleia-ft.el

```diff
@@ -243,41 +247,14 @@ environment variable, defaulting to all formatters."
                 (copy-to-buffer stdout-buffer (point-min) (point-max))))
           (progn
 
+            (let ((result (apheleia--format-command command nil nil)))
+              (setq command (nthcdr 3 result)
+                    in-temp-real-file (nth 0 result)
+                    out-temp-file (nth 1 result)))
```

Comment:
I'm not sure if there was a reason we didn't use this originally, I found with some of the later formatter updates I did it didn't substitute evaluations that produced nil or lists of values correctly. Rather than continually reverse engineering `apheleia--format-command` I opted to just call it directly :upside_down_face:.

Reply:
If it works then I'm fine with it. I suspect I didn't realize that the function was sufficiently self-contained and side-effect-free to be usable from tests. Or, it didn't exist when I wrote the test framework. Thanks.